data/reusables/desktop/fork-type-prompt.md

```diff
@@ -0,0 +1 @@
+1. If you clone a repository that you don't have write access to, {% data variables.product.prodname_desktop %} will create a fork. After creating or cloning a fork, {% data variables.product.prodname_desktop %} will ask how you are planning to use the fork. Click **To contribute to the parent project** if you plan to use this fork for contributing to the original upstream repository or click **For my own purposes** if you plan to use this fork for a project not connected to the upstream. Then click **Continue**.
```

Comment:
This makes sense and I see how it'll help people understand the procedures. The forking process only happens at the end of the cloning flow, so the "Forking a repository" section would have the entire "Cloning a repository" section and then the forking information. Is this going to be too redundant? I made this change in the latest commit if you want to see it in staging.

Reply:
> Is this going to be too redundant? I made this change in the latest commit if you want to see it in staging.

It would be redundant for anyone reading the entire article. But my guess is that someone opens the article, reads about the difference between forking and cloning, and then only reads the procedural that applies to their use case. We start each procedural from the very beginning because we can't assume folks have read the procedures above. 😊